payment-request/payment-request-abort-method.https.html

```diff
@@ -43,7 +43,7 @@
   try {
     await request.abort();
   } catch (err) {
-    assert_true(false, "Unexpected promise rejection: " + err.message);
```

Comment:
In these cases, it would be even better to remove the try/catch entirely, so that we can get the error appropriately propagated to the test harness (with stack trace etc.)